test/apps/mapbox-extension/README.md

```diff
@@ -0,0 +1,20 @@
+Integrations with the @deck.gl/mapbox extension
```

Comment:
It's technically not an extension, can we rename this directory to `mapbox-integration` or something similar?